projects/plugins/boost/app/modules/optimizations/page-cache/pre-wordpress/Boost_Cache.php

```diff
@@ -54,6 +54,38 @@ public function init_actions() {
 		add_action( 'switch_theme', array( $this, 'delete_cache' ) );
 		add_action( 'wp_trash_post', array( $this, 'delete_on_post_trash' ), 10, 2 );
 		add_filter( 'wp_php_error_message', array( $this, 'disable_caching_on_error' ) );
+
+		/**
+		 * Delete all cache.
+		 *
+		 * Allow third-party plugins to clear all cache.
+		 */
+		add_action( 'clear_page_cache_all', array( $this, 'delete_cache' ) );
+
+		/**
+		 * Delete cache for homepage and paged archives.
+		 *
+		 * Allow third-party plugins to clear front-page cache.
+		 */
+		add_action( 'clear_page_cache_home', array( $this, 'delete_cache_for_front_page' ) );
+
+		/**
+		 * Delete cache for a specific URL.
+		 *
+		 * Allow third-party plugins to clear the cache for a specific URL.
+		 *
+		 * @param string $url - The URL to delete the cache for.
+		 */
+		add_action( 'clear_page_cache_url', array( $this, 'delete_cache_for_url' ) );
+
+		/**
+		 * Delete cache for a specific post.
+		 *
+		 * Allow third-party plugins to clear the cache for a specific post.
+		 *
+		 * @param int $post_id - The ID of the post to delete the cache for.
+		 */
+		add_action( 'clear_page_cache_post', array( $this, 'delete_cache_by_post_id' ) );
```

Comment:
Bye bye dear generic action!

```suggestion
		add_action( 'jetpack_boost_clear_page_cache_post', array( $this, 'delete_cache_by_post_id' ) );
```